index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+     <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <div class="container">
+            <div>
+                <a class="mylogo" href="#top">My Logo</a>
+            </div>
+            <nav class="menuholder">
+                <img src="src/menuicon.png" alt="test">
+    
+            </nav>
+        </div>
+    </header>
+    
+    <section class="container2">
+        <div class="headerholder headmargin"> 
+            <h1 class="head">I’m Mariane<br>Glad to see<br> you!</h1>
+        </div>
+        <p class="headtext headmargin">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+
+        <div class="supporttext headmargin">
+            <b class="suptext">
+                LET'S CONNECT
+            </b>
+        </div>
+        
+        <ul class="social"> 
+            <li >
+                <img src="src/Vector.svg" alt="">
+            </li>
+            <li>
+                <img src="src/Linkedinicon.svg" alt="">
+            </li>
+            <li>
+                <img src="src/Vector.png" alt="">
+            </li>
+            <li>
+                <img src="src/Vector2.svg" alt="">
+            </li>
+            <li>
+                <img src="src/Vector3.svg" alt="">
+            </li>
```

Comment:
- Please, kindly update your social media icons to be wrapped in an anchor tag to make them a link.
`<li><a href=""><img ........></a></li>`